vendor/bat-native-ads/src/bat/ads/internal/tab_manager/tab_info.cc

```diff
@@ -14,4 +14,16 @@ TabInfo::TabInfo(
 
 TabInfo::~TabInfo() = default;
 
+bool TabInfo::operator==(
```

Comment:
@tmancey overloaded operator doesn't seem to be used, is it still good practice to define?

Reply:
Good spot I will remove these